AGGREGATE.md

```diff
@@ -238,13 +242,13 @@ utilize techniques like retries to minimize data loss.
   
 * The `privacy_budget_key` is used to define distinct batches of aggregate
   reports. It is used by the aggregation service to prevent replay attacks. It
-  will be a hash of: `reporting_origin | source_site | destination | version`.
+  will be a hash of:
+  `reporting_origin | source_site | destination | version | source_registration_time`.
   Note that the true key used to track batches will be `privacy_budget_key`
-  concatenated with `round_to_hour(scheduled_report_time)` and 
-  `source_registration_time`. The latter two are omitted from the key
-  for ergonomic reasons to help keys stay consistent over time. All reports that
-  share a privacy budget key should be sent to the aggregation service at the same 
-  time (in any order).
+  concatenated with `round_to_hour(scheduled_report_time)`. The latter is
+  omitted from the key to allow server-side recording to be time-bounded. All
+  reports that share a privacy budget key should be sent to the aggregation
```

Comment:
"All reports that share a privacy budget key should be sent to the aggregation..." should this mention that the privacy budget key here refers to the "true key"?

Reply:
Added a slight clarification here